ui/app/components/secure-variable-form.hbs

```diff
@@ -15,23 +16,42 @@
     </div>
     {{#each this.keyValues as |entry|}}
     <div class="key-value">
+
       <label>
         <span>Key</span>
         <Input
           @type="text"
           @value={{entry.key}}
-          class="input" />
+          class="input"
+          />
       </label>
+
       <label>
-        <span>Value</span>
+        <span>Value
+        </span>
         <Input
-          @type="text"
+          @type={{this.valueFieldType}}
           @value={{entry.value}}
-          class="input" />
+          class="input value-input"
+          autocomplete="new-password" {{!-- prevent auto-fill --}}
+          />
       </label>
+
+      <button
+        class="show-hide-values button is-borderless"
+        type="button"
+        tabindex="-1"
+        {{on "click" this.toggleShowHide}}
+      >
+        <FlightIcon
+          @name="{{if this.hideValues "eye-off" "eye"}}"
+          @title="{{if this.hideValues "Show Values" "Hide Values"}}"
```

Comment:
`question` `non-blocking`: Shouldn't we remove the quotations since these are dynamic values?